src/test/java/com/google/collegeplanner/TestCourse.java

```diff
@@ -0,0 +1,41 @@
+package com.google.collegeplanner.data;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests Course */
+@RunWith(JUnit4.class)
+public final class TestCourse {
+  private String[] gradingSystem;
+  private Course course;
+
+  @Before
+  public void before() {
+    String[] gradingSystem = new String[] {"Regular", "Pass-Fail"};
+    Course course = new Course("CMSC101", "Introduction to Computer Science", "202008", 4, "CMSC",
```

Comment:
you need to do `this.course = new Course(...);`

Otherwise, `this.course` will be `null` since it is never initialized.